example/views.py

```diff
@@ -4,12 +4,22 @@
 from example.serializers import (
     BlogSerializer, EntrySerializer, AuthorSerializer, CommentSerializer)
 
+from rest_framework_json_api.utils import format_drf_errors
+
 
 class BlogViewSet(viewsets.ModelViewSet):
     queryset = Blog.objects.all()
     serializer_class = BlogSerializer
 
 
+class BlogCustomViewSet(viewsets.ModelViewSet):
+    queryset = Blog.objects.all()
+    serializer_class = BlogSerializer
+
+    def handle_exception(self, exc):
+        return format_drf_errors(super(BlogCustomViewSet, self).handle_exception(exc), self.get_exception_handler_context(), exc)
```

Comment:
Please split this into multiple lines, it's a bit long.

Also would you mind adding some comments to this function so devs reading this file for an example can easily understand that it is an optional override that enables JSON-API to work alongside the regular DRF format?